php_phongo.c

```diff
@@ -1390,8 +1390,6 @@ static mongoc_client_t *php_phongo_make_mongo_client(const mongoc_uri_t *uri, zv
 	const char                *mongoc_version, *bson_version;
 	mongoc_client_t           *client;
 
-	ENTRY;
```

Comment:
Yes. As this was the only file that failed to compile now @bjori made it not leak.